lib/npdu.js

```diff
@@ -2,6 +2,7 @@
 
 const baEnum      = require('./enum');
 
+const DEFAULT_HOP_COUNT = 255;
```

Comment:
The BACNET Standard (135-2016 - 6.2.3) defines the default value as `0xFF`, as it was before.

Reply:
My mistake, will fix this.

Reply:
This one is also part of PR #134 so I will apply the fix over there.  If you are happy to merge that one then I will rebase this one to pick up the changes.